libs/blocks/accordion/accordion.css

```diff
@@ -144,3 +144,82 @@ dl.accordion {
 .dark .accordion dt button:hover .accordion-icon::after {
   background: var(--color-gray-100);
 }
+
+/* Editorial Variation  */
+.accordion-media {
+  display:  none;
+}
+
+.accordion-media > div {
+  position: relative;
+  display: none;
+  animation-duration: .1s;
+  animation-name: slide-left;
```

Comment:
Is this needed? From doing some tinkering, if i comment this out nothing changes. [Line 180](https://github.com/adobecom/milo/pull/942/files#diff-7594a3270934cbcf25a5670e5dff1ab0f3ebac12f12745ecf9bfb2fe9127e97fR180) appears to be controlling.